frame/wormhole/issuing/s2s/Cargo.toml

```diff
@@ -45,12 +44,17 @@ ethereum     = { version = "0.11.1", features = ["with-codec"] }
 evm          = { version = "0.33.1", features = ["with-codec"] }
 libsecp256k1 = { version = "0.5", features = ["static-context", "hmac"] }
 rlp          = { version = "0.5" }
+rustc-hex    = { version = "2.1.0", default-features = false }
```

Comment:
Why need this?

Reply:
tried to use array_bytes to convert the hex string, but it failed since the string is too long.

Reply:
@AurevoirXavier please review or suggest.